src/routes/SQLScheduler/views/SQLSchedulerJobForm.tsx

```diff
@@ -0,0 +1,189 @@
+          },
+  });
+  const onSubmit: SubmitHandler<SQLJobInput> = async (data: SQLJobInput) => {
+    let result;
+    setShowLoader(true);
+    if (type === 'add') {
+      // CREATE
+      result = await apiPost(`${gcUrl}/api/scheduled-jobs/`, data, {
+        credentials: 'include',
+      });
+    } else {
+      // UPDATE
+      const job = props.job;
+      result = await apiPut(`${gcUrl}/api/scheduled-jobs/${job.id}`, data, {
+        credentials: 'include',
+      });
+    }
+
+    setShowLoader(false);
+    if (result.success) {
+      backToClusterView();
+    }
+  };
+
+  const cron = watch('cron');
+  const cronHumanReadable = useMemo(() => {
+    try {
+      return cronstrue.toString(cron);
+    } catch (e) {
+      return null;
+    }
+  }, [cron]);
+
+  const executeQuery = useCallback(() => {
+    const sql = watch('sql');
+    setQueryRunning(true);
+    setQueryResults(undefined);
+    executeSql(sqlUrl, sql).then(({ data }) => {
+      setQueryRunning(false);
+      if (data) {
+        setQueryResults(data);
+      }
+    });
+  }, []);
+
+  const renderResults = () => {
+    if (queryRunning) {
+      return <Loader />;
+    }
+    return <SQLResultsTable results={queryResults} />;
+  };
+
+  useEffect(() => {
+    register('sql', { required: true, value: type === 'edit' ? props.job.sql : '' });
+  }, [register]);
+
+  if (showLoader) {
+    return (
+      <div className="flex items-center justify-center w-full h-full">
+        <Loader size={Loader.sizes.LARGE} color={Loader.colors.PRIMARY} />
+      </div>
+    );
+  }
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)} className="flex flex-col gap-4">
+      <div className="flex gap-4">
+        <div className="flex flex-col grow gap-2">
+          <Input
+            {...register('name', { required: true, maxLength: 99 })}
+            label={'Job Name'}
+          />
+          {errors.name && <span>This field is required</span>}
+        </div>
+
+        <div className="flex flex-col grow gap-2">
+          <Input
+            {...register('cron', { required: true })}
+            label={'Schedule'}
+            placeholder="* * * * *"
+          />
+          {errors.cron && <span>This field is required</span>}
+        </div>
+
+        <div className="flex flex-col gap-2">
+          <Checkbox {...register('enabled')} label={'Active'} />
+        </div>
+      </div>
+
+      <div>
+        <SQLEditor
+          onExecute={() => {
+            executeQuery();
+          }}
+          results={queryResults}
+          value={type === 'edit' ? props.job.sql : ''}
+          onChange={query => {
+            setValue('sql', query, { shouldValidate: true });
+          }}
+        />
+        {errors.sql && <span>This field is required</span>}
+      </div>
+
+      <Text>
+        {cronHumanReadable ? (
+          <>
+            This job will run{' '}
+            <span className="font-bold">
+              {cronHumanReadable.toLocaleLowerCase()}
```

Comment:
Maybe a dot at the end to finish the sentence :) 

```suggestion
              {cronHumanReadable.toLocaleLowerCase()}.
```